secure-boot-example/README.md

```diff
@@ -1,8 +1,153 @@
-This directory contains an example secure boot image signed with the example private key in this directory.
+# Secure-boot quickstart
+
+## Overview
+Secure-boot is a mechanism for verifying the integrity of the kernel image and 
+other files required during boot by storing them in a signed disk image.
+These files include the GPU firmware (start.elf etc), Kernel, initrd, device-tree tree
+and overlays.
+Secure-boot does require any particular OS and does not extend into the OS
+running on the CPU.  However, since the firmware will only load trusted 
+kernel + initrd images the kernel/initrd can safely verify or mount an
+an encrypted file-system e.g. by using a TPM.
+
+N.B The memory for the secure ramdisk is reclaimed as soon as the CPU is started.
+
+## Example OS
+This example includes a very simple OS image for Compute Module 4 generated
```

Comment:
a very simple OS image for Compute Module 4 generated using buildroot -> reword?
a very simple OS image generated using buildroot for Compute Module 4

Reply:
done